hub/api/tests/test_api.py

```diff
@@ -1319,6 +1319,14 @@ def test_empty_extend(memory_ds):
     assert len(ds) == 0
 
 
+def test_extend_with_progressbar():
+    ds1 = hub.dataset("mem://ds1")
+    with ds1:
+        ds1.create_tensor("x")
+        ds1.x.extend([1, 2, 3, 4], progressbar=True)
+    np.testing.assert_array_equal(ds1.x, np.array([[1], [2], [3], [4]]))
```

Comment:
use `ds1.x.numpy()` to get the numpy array otherwise this test will fail. Please try running the test.

Reply:
Hey @FayazRahman! I ran the test before pushing, and it passed with no problems. Here's a screenshot of the test:
![image](https://user-images.githubusercontent.com/53207198/167171563-b2b9d92b-6b5c-4741-b954-4c20209a2388.png)

I just used the same logic as a previous test in the same file, `test_extend_with_tensor`, where two Hub tensors are compared in the same way: https://github.com/activeloopai/Hub/blob/f7d5bef8116090022b19e88bab0c80e6baa4ae3b/hub/api/tests/test_api.py#L1309

I understand your point, so while these two tests pass, do you want me to update both with the `.numpy()` function? Or will just the updation of the test I created be enough?

Reply:
Everything's good then!